bip-update-process.md

```diff
@@ -0,0 +1,595 @@
+community. Some BIPs may never be adopted. Some BIPs may be adopted by one or multiple implementations or even
+incorporated into the Bitcoin protocol.
+
+### What should be documented in a BIP?
+
+The BIPs repository is focused on information and technologies that support and expand the utility of the bitcoin
+currency.[^astroturfing]
+
+## BIP format and structure
+
+### Specification
+
+BIPs should be written in mediawiki or markdown format.
+
+Each BIP should have the following parts:
+
+* Preamble -- Headers containing metadata about the BIP (see the section [BIP header preamble](#bip-header-preamble)
+  below).
+* Abstract -- A short (~200 word) description of the technical issue being addressed.
+* Motivation -- The motivation is critical for BIPs. It should clearly explain what issue the BIP addresses, and how the
+  existing situation is inadequate to address the problem that the BIP solves.
+* Specification -- The technical specification should describe the syntax and semantics of any new feature. The
+  specification should be detailed enough to enable any Bitcoin project to create an interoperable implementation.
+* Rationale -- The rationale fleshes out the specification by describing what motivated the design and why particular
+  design decisions were made. It should describe alternate designs that were considered and related work. The rationale
+  should address important objections or concerns raised during discussion.
+* Backwards compatibility -- BIPs must include a section describing any backwards incompatibilities and their severity.
+  The BIP must explain how implementors/users should deal with these incompatibilities.
+* Reference implementation -- Where applicable, a reference implementation, test vectors and documentation must be
+  completed before the BIP can be given status "Preliminary". Any or all of these can be provided either in the BIP or as
+  auxiliary files. It is recommended to focus on specification and rationale before writing code.
+* Change Log -- An optional section to track modifications to a BIP after reaching Preliminary status.
+* Footnotes -- A collection of footnotes cited in the BIP, and a place to list non-inline hyperlink targets
+* Copyright -- The BIP must be placed under an acceptable license ([see BIP Licensing below](#bip-licensing)).
+
+Some sections may not be necessary for all BIPs. When in doubt, a section should be included and briefly state how its
+topic has been sufficiently addressed.
+
+#### BIP header preamble
+
+Each BIP must begin with an RFC 822 style header preamble. The headers must appear in the following order. Headers
+marked with "\*" are optional. All other headers are required.
+
+```
+  BIP: <BIP number, or "?" before being assigned>
+* Layer: <Consensus (soft fork) | Consensus (hard fork) | Peer Services | API/RPC | Applications>
+  Title: <BIP title; maximum 44 characters>
+  Author: <list of authors’ names and email addresses>
```

Comment:
Almost all of them are further explained below, is it possible that the "see below" is unnecessary when reading the document rather than reviewing the markdown?

Reply:
I added the hint "The overview is followed by explanations for each header."